Minecraft_BTE_Tips/index.html

```diff
@@ -0,0 +1,140 @@
+              <i class="fas fa-atlas"></i>&nbsp;Teams</a>
+          </li>
+          <li class="nav-item">
+            <a href="../FAQ/index.html" class="nav-link p-3" style="font-size: large;">
+              <i class="fas fa-question-circle"></i>&nbsp;FAQ</a>
+          </li>
+          <li class="nav-item">
+            <a href="../Minecraft_BTE_Tips/index.html" class="nav-link p-3" style="font-size: large;">
+              <i class="far fa-lightbulb"></i>&nbsp;Minecraft BTE Tips</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+  <br>
+  <br>
+  
+
+  <section class="index-news">
+    
+    <h1 style="text-align: center;color: black;">Tips for ASEAN BTE Builders</h1>
+    <div class="container px-5 py-24 mx-auto">
+      <div class="flex items-center lg:w-3/5 mx-auto border-b pb-10 mb-10 border-gray-200 sm:flex-row flex-col">
+        <div class="flex-grow sm:text-left text-center mt-6 sm:mt-0">
+          <h2 style="color: black;",>Worldedit</h2><article>
+          <p style="color: black;">You all know of the traditional way of building right? Well, we didnt use that. Instead, we implemented a plugin named Worldedit. Worldedit is a plugin that can place blocks and replace them at will. Its basically your bread and butter!</p>
+        </div>
+      </div>
+    </article>
+    <br>
+    
+      <div class="flex items-center lg:w-3/5 mx-auto border-b pb-10 mb-10 border-gray-200 sm:flex-row flex-col">
+        <div class="flex-grow sm:text-left text-center mt-6 sm:mt-0">
+          <h2 style="color:black">Map of the world.</h2><article>
+        <p style="color: black;">You can also find other players on the world by pressing 'M'. By doing so, it pulls up a geographical map of the world and shows players location. If you are someone with special access, you can teleport to them!</p>
+            </svg>
+          </a>
+        </div>
+      </div>
+    </article>
+    <br>
+    
+      <div class="flex items-center lg:w-3/5 mx-auto sm:flex-row flex-col">
+        <div class="flex-grow sm:text-left text-center mt-6 sm:mt-0">
+          <h2 style="color: black;">Amenu</h2><article>
+          <p style="color: black;">A special thing we implemented is Amenu! Its a menu where you can warp/teleport to places and monuements!</p>
+            </svg>
+          </a>
+        </div>
+      </article>
+  </section>
+  
+  <div class="text-center my-3">
+    <a href="https://discord.gg/46Ed3UUJRD"><i class="footer_icon fab fa-discord"></i></a>
+    <a href="https://www.instagram.com/bte.asean/"><i class="footer_icon fab fa-instagram"></i></a>
+    <a href="https://github.com/ASEAN-Build-The-Earth/"><i class="footer_icon fab fa-github"></i></a>
+    <a href="https://www.youtube.com/c/TeamASEANBuildTheEarth/videos"><i class="footer_icon fab fa-youtube"></i></a>
+    <a href="https://twitter.com/bte_asean"><i class="footer_icon fab fa-twitter"></i></a>
+  </div>
+  <script src="./slidebar.js"></script>
```

Comment:
There is no use of this script in this page as there is no `header` tag
The `slidebar.js` is responsible for the fading of the header tag on scroll